Description: Re-adds missing keystone authtoken info in api-paste.ini
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2014-04-03

--- nova-2014.1~rc1.orig/etc/nova/api-paste.ini
+++ nova-2014.1~rc1/etc/nova/api-paste.ini
@@ -110,3 +110,15 @@ paste.filter_factory = nova.api.auth:Nov
 
 [filter:authtoken]
 paste.filter_factory = keystoneclient.middleware.auth_token:filter_factory
+auth_host = 127.0.0.1
+auth_port = 35357
+auth_protocol = http
+admin_tenant_name = %SERVICE_TENANT_NAME%
+admin_user = %SERVICE_USER%
+admin_password = %SERVICE_PASSWORD%
+# signing_dir is configurable, but the default behavior of the authtoken
+# middleware should be sufficient.  It will create a temporary directory
+# in the home directory for the user the nova process is running as.
+#signing_dir = /var/lib/nova/keystone-signing
+# Workaround for https://bugs.launchpad.net/nova/+bug/1154809
+auth_version = v2.0
